src/componenets/CardList.tsx

```diff
@@ -0,0 +1,28 @@
+import Card from "./Card";
+import "./CardList.css";
+
+interface CardListInterface {
+  id: number;
+  created_at: string;
+  updated_at: null;
+  url: string;
+  title: string;
+  description: string;
+  image_source: string;
+  folder_id: number;
+}
+
+type CardListProps = {
+  selectedFolderData: CardListInterface[];
+};
+
+function CardList({ selectedFolderData }: CardListProps) {
+  return (
+    <div className="cardList">
+      {selectedFolderData.map((cardData) => (
+        <Card key={cardData.id} data={cardData} />
+      ))}
```

Comment:
-Data라는 접미사도 지양해보면 좋을 것 같습니다~
```suggestion
      {selectedFolderData.map((card) => (
        <Card key={card.id} data={card} />
      ))}
```

Reply:
이렇게 표현할 수 있도록 Card 컴포넌트의 props에서 data없이 바로 card의 key값들을 받아주면 좋을 것 같아요!
```suggestion
      {selectedFolderData.map((card) => (
        <Card key={card.id} {...card} />
      ))}
```

현재는 이런데,
```ts
interface CardProps {
  data: {
    ...
 }
}
```

요렇게 하면 더 좋을 것 같아요
```ts
interface CardProps {
  ...
}
```

Reply:
@Seung-wan 이 부분은 어떻게 수정하면 좋은 지 감이 안잡혀요

Reply:
Card 컴포넌트에서 받는 props가 지금은 아래와 같이 되어있을탠데,
```ts
interface CardProps {
  data: {
    ...
  }
}
```

이렇게 바꿔주는 느낌입니다~
```ts
interface CardProps {
// data라는 key값 제거하기
  ...
}
```